src/docs/flex-cli/manage-search-schemas-with-flex-cli/index.md

```diff
@@ -111,6 +118,14 @@ flex-cli search set --key category --type enum --scope public -m my-test-marketp
 flex-cli search set --key amenities --type multi-enum --scope public -m my-test-marketplace
 ```
 
+Note that we don't need to supply `--schema-for` option when we add schema for `listing` as `listing` is the default.
+
+And we can also add a search schema for `userProfile` with `long` type.
+
+```shell
+flex-cli search set --schema-for userProfile --key age --type long --scope protected -m my-test-marketplace
+```
+
 We should now see the details of those schemas:
```

Comment:
If the `userProfile` schema is added as in the example, should it show in the schema details i.e. should the image `search-schemas-added.png` be replaced with one that shows the `userProfile`  schema as well?

Reply:
Good catch. I should first deploy some changes to staging so that I can take screenshot from my terminal. I will keep in mind.

Reply:
The image is replaced.